src/components/v5/shared/UserAvatarPopover/UserAvatarPopover.tsx

```diff
@@ -40,6 +46,17 @@ const UserAvatarPopover: FC<UserAvatarPopoverProps> = ({ size, ...props }) => {
       className={clsx({
         skeleton: !user,
       })}
+      additionalContent={
+        additionalContent || (!isVerified && showVerifiedBox) ? (
```

Comment:
If every place where the user popover is used must look the same (according to what Product wrote in the task at least), then I don't think we need this `showVerifiedBox` as the verified box should always show if the user is not verified.